src/targets/Logary.Targets.Serilog/Targets_Serilog.fs

```diff
@@ -0,0 +1,169 @@
+    | Value.Binary (bytes, _) -> toScalar bytes // TODO: content type? consider this again
+    | Value.Object map ->
+      let stringOrNull s = match s with String tt -> tt | _ -> null
+      let maybeTypeTag = map |> Map.tryPick (fun k v -> if k = "_typeTag" then Some (stringOrNull v) else None)
+      let typeTagOrNull = match maybeTypeTag with Some s -> s | None -> null
+      let serilogProperties =
+        map
+        |> Map.filter (fun k _ -> k <> "_typeTag")
+        |> Map.toSeq
+        |> Seq.map (fun (k, v) -> Serilog.Events.LogEventProperty(k, logaryValueToSerilogProperty v))
+
+      Serilog.Events.StructureValue(serilogProperties, typeTagOrNull)
+      :> Serilog.Events.LogEventPropertyValue
+    | Fraction (x, y) as f -> toScalar f // TODO:
+    | Value.Array items ->
+      Serilog.Events.SequenceValue (items |> List.map logaryValueToSerilogProperty)
+      :> Serilog.Events.LogEventPropertyValue
+
+  let logaryPointNameToSerilog (pointName : PointName) = seq {
+    yield Serilog.Events.LogEventProperty(
+      "SourceLoggerName", Serilog.Events.ScalarValue(pointName.ToString())) }
+
+  let defaultMessageToSerilogProperties (conf : SerilogConf) (logaryMessage : Message) =
+    let newSerilogProperty name value =
+      Serilog.Events.LogEventProperty(name, logaryValueToSerilogProperty value)
+
+    seq {
+      if conf.nameToSerilog.IsSome then
+        yield! conf.nameToSerilog.Value logaryMessage.name
+      
+      if not logaryMessage.fields.IsEmpty then
+        yield!
+          logaryMessage.fields
+          |> Map.toSeq
+          |> Seq.map (fun (pn, Field (v, u)) -> newSerilogProperty (string pn) v)
+
+      if not logaryMessage.context.IsEmpty then
+        yield!
+          logaryMessage.context
+          |> Map.toSeq
+          |> Seq.map (fun (k, v) -> newSerilogProperty k v)
+    }
+
+  let logaryToSerilog (conf : SerilogConf) (logaryMessage : Logary.Message) =
+    let template = match logaryMessage.value with
+                   | Event template -> template
+                   | _ -> failwith "logaryToSerilog only supports PointValue.Event"
+
+    Serilog.Events.LogEvent(
+      timestamp = DateTimeOffset.ofEpoch logaryMessage.timestamp,
+      level = toSerilogLevel logaryMessage.level,
+      ``exception`` = extractFirstExceptionOrNull logaryMessage,
+      messageTemplate = Serilog.Parsing.MessageTemplateParser().Parse template,
+      properties = conf.messageToSerilog conf logaryMessage)
+
+  type State = { logger : Serilog.ILogger }
+
+  let serve (conf : SerilogConf)
+            (ri : RuntimeInfo)
+            (requests : RingBuffer<_>)
+            (shutdown : Ch<_>) =
+    if isNull conf.logger then
+      failwith "You must configure the Logary SerilogConf with a valid Serilog ILogger instance."
+
+    let rec loop (state : State) : Job<unit> =
+
+      Alt.choose [
+        shutdown ^=> fun ack -> job {
+          // do! Job.Scheduler.isolate (fun _ -> (state :> IDisposable).Dispose())
```

Comment:
No disposal required for the Serilog target? Do you want this target to support the configuring of:

 1) A sink from Serilog + lifetime
 2) A logger from Serilog - assumed no lifetime, from source line this comment's on?
 3) A Serilog instance?